src/Components/Cart-Card/Cart-Card.jsx

```diff
@@ -0,0 +1,68 @@
+import { useCart } from "../../Context/Cart-Context/cart-context";
+import "./cart-card.css";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+export const CartCard = () => {
+  const { state, dispatch } = useCart();
+  const { cart } = state;
+  const notifyRemoveFromCart = () =>
+    toast.error("Item Removed from Cart", {
+      position: "bottom-right",
+      theme: "colored",
+    });
+  const notifyMoveToWishList = () =>
+    toast.success("Item moved to WishList", {
+      position: "bottom-right",
+      theme: "colored",
+    });
+  return (
+    <div class="all-cart-products">
+      {cart.length === 0 ? <h1>Please add Items in Cart</h1> : null}
+      {cart &&
+        cart.map((item) => (
+          <div class="horizontal-card">
+            <div class="horizontal-image-container">
+              <img src={item.image} class="horizontal-card-image" alt="" />
+            </div>
+            <div class="card-data">
+              <div class="card-details-horizontal">
+                <div class="card-product-name">
+                  <div class="product-name">{item.name}</div>
+                  <div class="product-details">{item.description}</div>
+                </div>
```

Comment:
item can be destructured here. It will help in working and code readability too

Reply:
Actually I was about to do that , but in the payload I had to pass the whole item so changed my mind to leave it as it is